src/_posts/hands-on-i.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Hands-On! Building Interactive Training With A-Frame
+author: twitter|salvadelapuente|Salvador de la Puente
+date: 2017-08-08 12:00:00
+layout: blog
+image:
+  src: hands-on-i.png
+---
+
+In an industry where manipulation of complex mechanisms involves tens of tools, expensive equipment and even safety risks, virtual reality helps to increase operator safety and reduce material costs. In addition, WebVR puts everything just one link away.
```

Comment:
agreed, something like [this](https://tympanus.net/codrops/2016/01/12/cinema-seat-preview-experiment/) would be great